client/src/components/CityDropDownOptions.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+export default function CityDropDownOptions({ city, handleChange }) {
+  const cities = {
+    "North-West": [1, 2, 3, 4],
+    London: [1, 2, 3, 4, 5, 6, 7],
+    Scotland: [1, 2, 3, 4, 5],
+  };
+
+  const cityDropDown = () => {
+    if (city === "London") {
+      return (
+        <>
+          <label for="userClassId">Choose a Class</label>
+          <select name="userClassId" onChange={handleChange}>
+            <option value="select">Select</option>
+            {cities.London.map((cohort, index) => (
+              <option key={index} value={cohort}>
+                {cohort}
+              </option>
+            ))}
+          </select>
+        </>
+      );
+    }
```

Comment:
🟢 Each of these 3 `if` blocks look very similar, you _could_ extract the middle bit into a teeny tiny component, or you could get clever with a map, like your existing `Object.keys(cities).map(...)`
Or you could leave it as it is :D

Reply:
@petemcfarlane we tried the Object.keys(cities).map(...) option last night but couldn't figure it out. any hints?

Reply:
Something like
```js
Object.keys(cities).map((city) => (
  <>
    <label for="userClassId">Choose a Class</label>
    <select name="userClassId" onChange={handleChange}>
      <option value="select">Select</option>
      {cities[city].map((cohort, index) => (
        <option key={index} value={cohort}>
          {cohort}
        </option>
      ))}
    </select>
  </>
))
```